velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -1588,8 +1588,9 @@ Expected<DateTimeResult> DateTimeFormatter::parse(
     }
   }
 
-  // Ensure all input was consumed.
-  if (cur < end) {
+  // Ensure all input was consumed if type_ is not legacy datetime formatter.
```

Comment:
> legacy datetime formatter

Do we have a clear definition of which formatters are considered legacy?

Reply:
Thanks. I updated the comment to use `LENIENT_SIMPLE or STRICT_SIMPLE datetime formatter` instead of `legacy datetime formatter`.